book/actix-plugin.md

```diff
@@ -30,10 +30,15 @@ struct Pet {
 }
 
 async fn echo_pet(body: Json<Pet>) -> Result<Json<Pet>, ()> {
-    Ok(body)
+    let new_pet = Pet {
+        name: format!("The New {}", body.name),
+        id: body.id
+    };
```

Comment:
This really feels like an unnecessary change when we're trying to demo the plugin.

Reply:
I wanted to show it that the response should be a JSON for sure! (Not a HttpResponse or ...) 
And how to create it!

Or I'm wrong.

Reply:
No, it's just that it's an echo handler, so why not respond with the body as it is?

Reply:
OK, done